Resources/public/js/views/fields/ez-relationlist-editview.js

```diff
@@ -93,14 +93,14 @@ YUI.add('ez-relationlist-editview', function (Y) {
         },
 
         /**
-         * Fire the `loadObjectRelations` event
+         * Fire the `loadFieldRelations` event
          *
          * @method _fireLoadObjectRelations
          * @protected
          */
         _fireLoadObjectRelations: function () {
             if ( !this._isFieldEmpty() ) {
-                this.fire('loadObjectRelations', {
+                this.fire('loadFieldRelations', {
```

Comment:
you should not change the event name first because we should avoid to do that in a minor release second because the info that you want the relation in a given field is already expressed in the event parameters.